core/helpers/index.ts

```diff
@@ -159,3 +160,10 @@ export function once (key, fn) {
 }
 
 export const isServer: boolean = typeof window === 'undefined'
+
+// Online/Offline helper
+export const onlineHelper = Vue.observable({ 
```

Comment:
should be just isOnline like isServer imho por teh sake of consistance

Reply:
yeah, but then it would be `isOnline.isOnline` which would be weird usage